mlflow/utils/requirements_utils.py

```diff
@@ -271,22 +269,56 @@ def _capture_imported_modules(model_uri, flavor):
         # resolution in a subprocess based on PATH entries.
         main_env["PATH"] = "/usr/sbin:/sbin:" + main_env["PATH"]
 
-        _run_command(
-            [
-                sys.executable,
-                _capture_modules.__file__,
-                "--model-path",
-                local_model_path,
-                "--flavor",
-                flavor,
-                "--output-file",
-                output_file,
-                "--sys-path",
-                json.dumps(sys.path),
-            ],
-            timeout_seconds=process_timeout,
-            env=main_env,
-        )
+        if flavor == mlflow.transformers.FLAVOR_NAME:
+            # Lazily import `_capture_transformers_module` here to avoid circular imports.
+            from mlflow.utils import _capture_transformers_modules
+
+            for module_to_throw in ["tensorflow", "torch", ""]:
```

Comment:
```suggestion
            for module_to_throw in ["torch", "tensorflow"]:
```

Can we swap torch and tf because transformers uses torch by default?

Reply:
Never mind this comment.